docs/sites.yml

```diff
@@ -5136,3 +5136,14 @@
   built_by: Ariadne Digital
   built_by_url: https://www.ariadnedigital.it
   featured: false
+  - title: OneDeck
```

Comment:
```suggestion
- title: OneDeck
```

Fixing the yaml formatting

Reply:
Thanks @lannonbr!

Reply:
If you agree with the fix, click that "commit suggestion" button